src/euphorie/client/docx/compiler.py

```diff
@@ -447,7 +454,20 @@ def add_measure(self, doc, heading, measure, implemented=False):
             ]
 
         m = measure
-        values = [_simple_breaks(m.action or "")]
+        action = m.action
```

Comment:
To reduce the lines of code of this method and to save an `if` in an already complex method, what about having something like:
```
action = self.measure2action(measure)
```

On the base class `measure2action` returns `measure.action` but for the others it returns the code in the if block.

I hope you get the point.